### PR TITLE
Best practices Review

### PR DESCRIPTION
**Pull request by maov19**
Added features:
- Branch created to review JS best practices.

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+const deleteBtn = document.querySelector('.delete');
+
+let taskList = [];
+
+const display = () => {
+  taskListDisplay.innerHTML = '';
+  taskList = JSON.parse(localStorage.getItem('localItem')) || [];
+  taskList.forEach((element) => {
+    const task = document.createElement('div');
+    const comp = element.completed ? 'checked' : '';
```
Comment (ernestmusong) [inline src/crud.js]: I would suggest you name your variables in a way that they reveal the intention behind it. You could rename **const comp** to **const completed**

File: src/crud.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    completed: false,
+  };
+  taskList.push(object);
+  localStorage.setItem('localItem', JSON.stringify(taskList));
+
+  inputValue.value = '';
+  display();
+});
+
+const remove = (index) => {
```
Comment (ernestmusong) [inline src/crud.js]: I would suggest you name your variables in a way that they reveal the intention behind it. You could rename **const remove** to **const removeTodo**

File: src/clear.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  if (!click) return;
+
+  const dataT = parseInt(click.getAttribute('data-set'), 10);
+  const task = JSON.parse(localStorage.getItem('localItem')) || [];
+  const find = task.find((todo) => todo.index === dataT);
+  find.completed = !find.completed;
+  localStorage.setItem('localItem', JSON.stringify(task));
+};
+
+export const clear = () => {
```
Comment (ernestmusong) [inline src/clear.js]: I would suggest you name your variables in a way that they reveal the intention behind it. You could rename **const clear** to const **clearCompleted**

File: src/clear.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+export const clearing = (e) => {
+  const click = e.target.closest('.check');
+  if (!click) return;
+
+  const dataT = parseInt(click.getAttribute('data-set'), 10);
```
Comment (ernestmusong) [inline src/clear.js]: I would suggest you name your variables in a way that they reveal the intention behind it. Please rename **const dataT**

File: src/clear.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  const task = JSON.parse(localStorage.getItem('localItem')) || [];
+  const find = task.find((todo) => todo.index === dataT);
+  find.completed = !find.completed;
+  localStorage.setItem('localItem', JSON.stringify(task));
+};
+
+export const clear = () => {
+  const set = JSON.parse(localStorage.getItem('localItem')) || [];
```
Comment (ernestmusong) [inline src/clear.js]: I would suggest you name your variables in a way that they reveal the intention behind it. Please rename c**onst clear** and **const set**

**Review by ernestmusong — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @maov19 , 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔
-  Your code is DRY ✔

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crud.js (hunk 1 above)
Comment (maov19) [reply src/crud.js]: Done! 👍

File: src/crud.js (hunk 2 above)
Comment (maov19) [reply src/crud.js]: Done! 👍

File: src/clear.js (hunk 3 above)
Comment (maov19) [reply src/clear.js]: Done! Changed clear for clearCompleted 🔧

File: src/clear.js (hunk 4 above)
Comment (maov19) [reply src/clear.js]: Done! Replaced dataT for localCheck 🔧

File: src/clear.js (hunk 5 above)
Comment (maov19) [reply src/clear.js]: Done!
Replaced clear for clearCompleted 🔧 
Replaced set for getLocal 🔨

**Comment by maov19** [comment]
Hi @ernestmusong!
Thanks for your feedback 👍 
All the mentioned issues were solved. Please let me know if there's something else I should work on.

**Review by ernestmusong — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @maov19  👋, 

Well done for implementing the required changes 👏 
Your project is now complete!  And There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**